.github/workflows/ci-core-reusable.yml

```diff
@@ -120,6 +120,7 @@ jobs:
   integration:
     name: Integration (consensus=${{ matrix.consensus }})
     strategy:
+      fail-fast: false
```

Comment:
Leftover (here and below)?
We can, in theory, discuss whether we want to disable fail-fast in CI, but I'd expect it to be done separately and with a prior discussion internally.

Reply:
done